runtime/lua/_vim9script.lua

```diff
@@ -29,7 +29,7 @@ local vim9 = (function()
 
   M.fn_ref = function(module, name, copied, ...)
     for _, val in ipairs({ ... }) do
-      table.insert(copied, val)
+      copied[#copied + 1] = val
```

Comment:
Would this need to be upstreamed to https://github.com/tjdevries/vim9jit ? @tjdevries

Reply:
yeah, that would be the place to put that. @glepnir if you want to make an issue, i'll fix it and can submit PR with updates & generated code